VideoLocker/res/values-en/errors.xml

```diff
@@ -38,4 +38,10 @@
     <string name="video_offline_header">This video is not available offline.</string>
     <string name="video_offline_message">Please select a video that you\'ve downloaded onto your device.</string>
 
-</resources>
\ No newline at end of file
+    <!-- Registration Error Messages -->
+    <string name="error_enter_field">Please enter your %1$s.</string>
+    <string name="error_select_field">Please select your %1$s.</string>
+    <string name="error_min_length">%1$s must have at least %2$d characters.</string>
+    <string name="error_max_length">%1$s cannot have more than %2$d characters.</string>
```

Comment:
As per this http://developer.android.com/guide/topics/resources/string-resource.html#FormattingAndStyling we need to add <code>$</code>. As the % symbol is not escaped when used in xml, We can either use this format or add <code>format="false"</code> to all the strings.